### PR TITLE
add latest to supported automate version

### PR DESCRIPTION
**Pull request by susanev**
### Description

[internal slack convo](https://chefio.slack.com/archives/C07HSSZ6U/p1601485980042500) requested that we say we only support latest automate

### Definition of Done
stuff is better documented

### Issues Resolved
zero.

### Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: content/versions.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ announcement](https://blog.chef.io/2019/04/02/chef-software-announces-the-enterp
 <tbody>
 <tr class="odd">
 <td>Chef Automate</td>
-<td>2</td>
+<td>Version 2 Latest build</td>
```
Comment (kagarmoe) [inline content/versions.md]:
```suggestion
<td>Latest</td>
```

Do we need to disambiguate between versions here?

Comment (susanev) [reply content/versions.md]: was thinking cause of automate 1, but maybe not? i can update it to just latest if yall want

Comment (kagarmoe) [reply content/versions.md]: Thanks! 1.x is in the EOL table.

**Review by kagarmoe — APPROVED** [review]
tenor-157475859

**Pull request merged**